cmd/thor/node/metrics.go

```diff
@@ -16,4 +16,6 @@ var (
 	metricBlockProcessedDuration = metrics.LazyLoadHistogram("block_processed_duration_ms", metrics.Bucket10s)
 	metricChainForkCount         = metrics.LazyLoadCounter("chain_fork_count")
 	metricChainForkSize          = metrics.LazyLoadGauge("chain_fork_size")
+	metricsGasPerBlock           = metrics.LazyLoadGauge("gas_per_block_gauge")
+	metricsTxsPerBlock           = metrics.LazyLoadGauge("txs_per_block_gauge")
```

Comment:
Looks like  `metricsGasPerBlock` and `metricsTxsPerBlock` are the gauge version of `metricBlockProcessedTxs` and `metricBlockProcessedGas`. What if the `metricBlockProcessedTxs` and `metricBlockProcessedGas` were set as Gauge ?

It seems like they are very similar and that the `metricBlockProcessedXXX` also have a label indicating if it was a block produced or received.

Reply:
Pushed this commit @otherview 

https://github.com/vechain/thor/pull/778/commits/4a1f1755dd2abcfa86a38bcaa2cd0e22d8578138